packages/react-native-windows-init/src/Cli.ts

```diff
@@ -83,14 +83,14 @@ export const windowsInitOptions = initOptions({
     type: 'string',
     describe: 'The type of project to initialize (supported on 0.64+)',
     choices: ['app', 'lib'],
-    default: 'app',
+    default: undefined,
   },
   experimentalNuGetDependency: {
     type: 'boolean',
     describe:
       '[Experimental] change to start consuming a NuGet containing a pre-built dll version of Microsoft.ReactNative',
     hidden: true,
-    default: false,
+    default: undefined,
```

Comment:
IIRC, the args parsing code won't ever actually return `undefined` at runtime with a type set to `'boolean'`. It will just return `false`. So setting this to undefined here is really misleading for devs, - which is why the tests fail - to stop devs from setting this.

Setting default to undefined only works for `string` options.